launch/planning_scene/find_planning_scene_boxes_helper.launch

```diff
@@ -3,9 +3,6 @@
 
   <!-- rosbag full path -->
   <arg name="bag_path" default="" />
-  <arg name="world_config" default="moelk_tables" doc="objects arrangement, options: moelk_tables, cic_tables, truck_assembly" />
-  <arg name="yaml_path_to_read" default="$(find tables_demo_bringup)/config/$(arg world_config)_planning_scene.yaml" />
-  <arg name="yaml_path_to_write" default="$(find tables_demo_bringup)/config/$(arg world_config)_planning_scene.yaml" />
```

Comment:
Why not handle this identically to launch/planning_scene/planning_scene_visualization.launch?

Reply:
The example planning scene originates from an APRIL use case and I didn't thought it would be a good default:

![Screenshot from 2024-04-12 13-11-50](https://github.com/DFKI-NI/grasplan/assets/49768113/e891fc81-ca1d-46f8-97d6-e11640b4c7a1)

Also it's specified in the `world` frame and not `map`. But after writing this, I think it would be best to change the default planning scene. Maybe just copy the `cic` planning scene from https://github.com/DFKI-NI/mobipick_labs/blob/noetic/tables_demo_bringup/config/cic_tables_planning_scene.yaml ?

Reply:
What I meant with my comment above was to do the identical change here as in the other launch file.

Whether to replace the example planning scene with the `cic_tables` planning scene is a different question. I would agree that it's a good idea. @oscar-lima: What do you think?

Reply:
Yeah sure, I think the old one comes from APRIL project, so we can safely change it since the project is over.